app/views/play/level/modal/CourseVictoryComponent.vue

```diff
@@ -152,7 +152,9 @@
         else
           { 'btn-success': true , 'btn-glow': true  }
       headerImage: ->
-        if @level.assessment
+        if @course._id is utils.courseIDs.COCO_JUNIOR
```

Comment:
will @course be undefined ever?

Reply:
it's coursevictorycomponent, so course should exist